packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -162,9 +171,11 @@ export async function rateLimitDateWhenUserNextAbleToPost(user: DbUser): Promise
   if (shouldIgnorePostRateLimit(user)) return null;
   
   // does the user have a moderator-assigned rate limit?
-  const [modRateLimitHours, userPostRateLimit] = await Promise.all([
+  // also get the recent karma info, we'll need it later
+  const [modRateLimitHours, userPostRateLimit, recentKarmaInfo] = await Promise.all([
     getModRateLimitHours(user._id),
-    getUserRateLimit(user._id, 'allPosts')
+    getUserRateLimit(user._id, 'allPosts'),
+    getRecentKarmaInfo(user._id)
```

Comment:
Just some performance improvement by getting the recentKarmaInfo at the same time as everything else